theme/static/css/hydroshare_core.css

```diff
@@ -97,10 +97,49 @@ h1.hl small, h2.hl small, h3.hl small, h4.hl small, h5.hl small, h6.hl small {
   background-color: #eaeaea;
 }
 
+.navbar-toggle{
+    background: #B5B5B5;
+    border-color: #8C8C8C !important;
+}
+
+/* ===== Recaptcha ===== */
+.fit{
+    max-width: 100%;
+}
```

Comment:
This isn't recaptcha specific. Is there a section of the CSS with general layout styles this could be added to?